src/app/course/course.component.html

```diff
@@ -1,92 +1,136 @@
-<table class="table table-bordered table-light mt-5 hover-shadow">
```

Comment:
I will need to take a deeper look at this file, in particular, to ensure it has intended behaviour.